jobs/ci-kubernetes-e2e-gce-1-6-master-upgrade-cluster.env

```diff
@@ -0,0 +1,12 @@
+### job-env
```

Comment:
I'm not super sure what are in Feature:MasterUpgrade and Feature:ClusterUpgrade.. @krousey maybe you can give us a short explanation then I'll update the comment.